code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -13,29 +13,32 @@
 		list(mode_name="semiauto",       burst=1, fire_delay=0,    move_delay=null, burst_accuracy=null, dispersion=null),
 		list(mode_name="3-round bursts", burst=3, fire_delay=null, move_delay=4,    burst_accuracy=list(0,-15,-15), dispersion=list(0.0, 0.6, 1.0)))
 
-/obj/item/weapon/gun/projectile/automatic/saber //Fixed it
-	name = "prototype SMG"
-	desc = "A protoype lightweight, fast firing gun. Uses 9mm rounds."
+/obj/item/weapon/gun/projectile/automatic/advanced_smg
+	name = "advanced SMG"
+	desc = "An advanced submachine gun with a reflective laser optic that makes burst fire less inaccurate than other SMGs. Uses 9mm rounds."
 	icon = 'icons/obj/gun.dmi'
-	icon_state = "saber"//Still ugly
+	icon_state = "advanced_smg-empty"

```

Comment:
See my comment above. I need to correct the sprites, but since I have like three things that muck with the gun.dmi file in the pipeline, I'm trying to work out first what stuff is going to get accepted and what won't.

Otherwise, each of these PRs will conflict with one-another. It'll save some headache and lessens risks of breaking icons if I can do all the icon updates at once after knowing what stays and what goes.

Reply:
Actually, I'm just going to nix the sprite changes to the tommygun PR as they aren't necessary.

So it's just going to be this thing and the G11 coming in a PR I plan to redo today.

Reply:
https://github.com/PolarisSS13/Polaris/pull/5886 makes icon changes that would cause conflicts in this PR, or vise-versa.

Reply:
Screw it I'll just change them now and fix conflicts later.